comp/logs/agent/config/endpoints.go

```diff
@@ -37,9 +37,9 @@ type Endpoint struct {
 	APIKey                  string `mapstructure:"api_key" json:"api_key"`
 	Host                    string
 	Port                    int
-	UseSSL                  bool
-	UseCompression          bool `mapstructure:"use_compression" json:"use_compression"`
-	CompressionLevel        int  `mapstructure:"compression_level" json:"compression_level"`
+	UseSSL                  *bool `mapstructure:"logs_no_ssl" json:"logs_no_ssl"`
```

Comment:
```suggestion
	UseSSL                  *bool `mapstructure:"use_ssl" json:"use_ssl"`
```
I think this should be `use_ssl`. `logs_no_ssl` is the inverse of the underlying setting. 

I it may be safe to rename since it never worked to begin with (as a public config), but I think we should probably leave it as is.